### PR TITLE
override org.apache.lucene

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: 1 [hunk 1]
```diff
@@ -0,0 +1,2691 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (porcelli) [inline 1]: @mbiarnes this file looks wrong here, could you please review?

Comment (mbiarnes) [reply 1]: @porcelli removed

**Comment by porcelli** [comment]
@mbiarnes just noticed that @cristianonicolai just sent a PR about it too:

https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/873

**Comment by mbiarnes** [comment]
Jenkins please retest this

**Comment by mbiarnes** [comment]
@porcelli @cristianonicolai 
All org.apache.lucene dependencies are in jboss-ip-bom. We only need to override the version. 
I will merge the PR https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/874 and close https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/873

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by cristianonicolai** [comment]
@mbiarnes @porcelli I'm checking if the appformer compilation issue is fixed with https://github.com/kiegroup/appformer/pull/514

**Comment by pefernan** [comment]
Jenkins please retest this

**Comment by mbiarnes** [comment]
will close this as the another PR #876 did the same

**Pull request closed without merging**